### PR TITLE
Master: DCOS-21229: fix(TaskTable): fix sorting in tables

### PR DESCRIPTION
**Pull request by DanielMSchmidt**
Closes DCOS-21229
Backport: https://github.com/dcos/dcos-ui/pull/2737

**Checklist**
- [X] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [X] Did you add new integration tests?
- [X] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: plugins/services/src/js/utils/TaskTableUtil.js [hunk 1]
```diff
@@ -3,6 +3,7 @@ import React from "react";
 /* eslint-enable no-unused-vars */
 import TableUtil from "#SRC/js/utils/TableUtil";
 import Util from "#SRC/js/utils/Util";
+import TaskUtil from "#PLUGINS/services/src/js/utils/TaskUtil";
```
Comment (juliangieseke) [inline plugins/services/src/js/utils/TaskTableUtil.js]: 😢

**Comment by DanielMSchmidt** [comment]
run integration tests

**Comment by nLight** [comment]
run integration tests

**Comment by DanielMSchmidt** [comment]
You are right @nLight, looking into this 👍

**Comment by DanielMSchmidt** [comment]
@nLight the tests should be fixed now 👍

**Comment by DanielMSchmidt** [comment]
run integration tests

**Comment by DanielMSchmidt** [comment]
@juliangieseke can you reapprove? 🙈

**Comment by juliangieseke** [comment]
nope :trollface:

**Pull request merged**